pages/_document.tsx

```diff
@@ -19,6 +19,12 @@ class MyDocument extends Document {
   }
 
   render(): React.ReactElement {
+    let tagManagerSrc= '';
+    if (process.env.GA_TRACKING_ID)
```

Comment:
I would double check that this works, I don't see `GA_TRACKING_ID` as being marked as a public environment variable and there is a variable with a different name configured in the next.config: https://github.com/uniformdev/uniformconf/blob/stable/next.config.js#L11